src/pages/Admin/Venue/VenueAdminPage.tsx

```diff
@@ -1,48 +1,47 @@
-import React, { FC } from "react";
+import React from "react";
 
 import {
-  isCurrentVenueRequestedSelector,
-  isCurrentVenueRequestingSelector,
-  venueSelector,
+  isCurrentVenueNGRequestedSelector,
+  isCurrentVenueNGRequestingSelector,
 } from "utils/selectors";
 
 import { IFRAME_TEMPLATES } from "settings";
 
 import { useSelector } from "hooks/useSelector";
-import { useUserIsVenueOwner } from "hooks/useUserIsVenueOwner";
-import useConnectCurrentVenue from "hooks/useConnectCurrentVenue";
+import { useIsUserVenueOwner } from "hooks/useIsUserVenueOwner";
 import { useUser } from "hooks/useUser";
+import { useVenueId } from "hooks/useVenueId";
+import { useConnectCurrentVenueNG } from "hooks/useConnectCurrentVenueNG";
 
-import { LoadingPage } from "components/molecules/LoadingPage/LoadingPage";
+import { LoadingPage } from "components/molecules/LoadingPage";
 import { AdminVideo } from "components/molecules/AdminVideo";
-import { AdminBanner } from "components/molecules/AdminBanner/AdminBanner";
+import { BannerAdmin } from "components/organisms/BannerAdmin";
 
 import "./VenueAdminPage.scss";
 
-export const VenueAdminPage: FC = () => {
-  useConnectCurrentVenue();
-
+export const VenueAdminPage: React.FC = () => {
   const { profile, user } = useUser();
-  const venue = useSelector(venueSelector);
-  const venueRequestStatus = useSelector(isCurrentVenueRequestedSelector);
-  const venueRequestingStatus = useSelector(isCurrentVenueRequestingSelector);
+  const venueId = useVenueId();
+  const { currentVenue: venue } = useConnectCurrentVenueNG(venueId);
```

Comment:
Probably would have renamed this variable throughout the code, as `currentVenue` is more descriptive of what it relates to.